meilisearch-http/src/index/mod.rs

```diff
@@ -83,6 +83,18 @@ impl Index {
             })
             .transpose()?
             .unwrap_or_else(BTreeSet::new);
+
+        let synonyms = self
+            .synonyms(&txn)?
+            .iter()
+            .map(|(key, values)| {
+                (
+                    key.join(" "),
+                    values.iter().map(|value| value.join(" ")).collect(),
+                )
+            })
+            .collect();
```

Comment:
why are we padding the key and values with whitspace? can you add a comment, so we don't forget?

Reply:
I wrote this comment:
```
        // in milli each word in the synonyms map were split on their separator. Since we lost
        // this information we are going to put space between words.
```